analyzers/src/SonarAnalyzer.Common/Helpers/PropertiesHelper.cs

```diff
@@ -43,18 +43,30 @@ public static XElement[] ParseXmlSettings(SourceText sourceText)
         public static bool ReadIgnoreHeaderCommentsProperty(IEnumerable<XElement> settings, string language) =>
             ReadBooleanProperty(settings, language, "ignoreHeaderComments");
 
-        private static bool ReadBooleanProperty(IEnumerable<XElement> settings, string language, string propertySuffix, bool defaultValue = false)
-        {
-            var propertyLanguage = language == LanguageNames.CSharp ? "cs" : "vbnet";
-            var propertyName = $"sonar.{propertyLanguage}.{propertySuffix}";
-            return settings.Any()
-                && GetPropertyStringValue(propertyName) is { } propertyStringValue
+        public static string[] ReadSourceFileInclusionsProperty(IEnumerable<XElement> settings, string language) =>
+            ReadCommaSeparatedArrayProperty(settings, language, "sonar.inclusions");
+
+        public static string[] ReadSourceFileExclusionsProperty(IEnumerable<XElement> settings, string language) =>
+            ReadCommaSeparatedArrayProperty(settings, language, "sonar.exclusions");
```

Comment:
The property name is wrong. With the current logic the property name will be `sonar.cs.sonar.exclusions` for C# ad `soanr.vbnet.sonar.exclusions` for VB.NET while it should be only `sonar.exclusions`.